### DISCUSSION TITLE
Welcome to EkiLib Discussions!

### DISCUSSION THREAD

**Discussion by ChAoSUnItY** [ChAoSUnItY/EkiLib#2] [General]
## 👋 Welcome to Eki Lib's Discussion!
  This is where you can:
  * Ask for some features I didn't explain very completely.
  * Share your ideas about future development.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Scwestbrook** [comment]
Hi, I'm Scwestbrook, and I mainly try to help out Minecraft modders with suggestions and ideas. This is my FAVORITE mod ever.
